release/RELEASE.md

```diff
@@ -207,7 +207,7 @@ latest commit on master.
 
        1. Run `daml build`.
 
-       1. In 3 separate terminals (since each command except for `daml script …`
+       1. In 5 separate terminals (since each command except for `daml script …`
```

Comment:
3 really are enough.
1. daml sandbox
2. daml script & daml navigator
3. daml codegen java & mvn compile exec:java@run-quickstart

Reply:
Ah, well, the parenthesis needs changing then.

Reply:
How does this look?

```
       1. In 3 separate terminals (since each command blocks), run:

          1. `daml sandbox --wall-clock-time --port 6865 .daml/dist/quickstart-0.0.1.dar`
          1. `daml script --dar .daml/dist/quickstart-0.0.1.dar --script-name Setup:initialize --ledger-host localhost --ledger-port 6865 --wall-clock-time && daml navigator server localhost 6865 --port 7500`
          1. `daml codegen java && mvn compile exec:java@run-quickstart`
```

Reply:
:+1: